### PR TITLE
Feature/add user manual to readme

### PR DESCRIPTION
**Pull request by Steffengreiner**
Description of changes
This PR introduces a user-manual into the Readme.md of the sample-tracking-update-portlet showcasing the current features with screenshots. 

Additional Information
Currently there are two ToDos included in this PR addressing unfinished sections in the user manual:

1.  An explanation on how to set the Sample Status has to be added to the user manual once the sample statuses are updated in the database. (See #43 for details)
2. Showcasing the updated arrival date of a sample is not possible in the sample-tracking-update portlet. How should the successful update be shown?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,88 @@ Created by all the [contributors](https://github.com/qbicsoftware/sample-trackin
...
+
+If the given sample ID is valid, the sample and its associated information will appear in the "Samples to update" grid 
+![add](readme-docs/screenshots/AddSampleByIdResult.png)
+
+Each subsequent Sample ID will be added to the "Samples to update" grid 
+![add](readme-docs/screenshots/AddSampleByIdAgain.png)
+ 
+#### Select Samples by File
+It is also possible to select the samples of interest by 
+uploading one or multiple *.csv files containing the sample codes in the first column.
```
Comment (jenniferboedker) [inline README.md:55]: maybe add an example here on how the CSV needs to look like, kind of a style guide

Comment (jenniferboedker) [reply README.md:55]: haha nvm found the example

Comment (jenniferboedker) [reply README.md:55]: new question, why all the commas in the example csv?

Comment (Steffengreiner) [reply README.md:55]: Good Catch! 👀 . 
Apparently Numbers exports csv with all columns even if they are empty. 
I updated the csv with only the set columns.

**Review by jenniferboedker — DISMISSED** [review]

**Comment by Steffengreiner** [comment]
I updated the Sample Status Section of the User Manual. Hopefully it's still up to par 👍

File: README.md [hunk 2]
```diff
@@ -11,10 +11,92 @@ Created by all the [contributors](https://github.com/qbicsoftware/sample-trackin
...
+
+After pressing the "Update Samples" button the status of all currently selected samples will be updated to the selected status.  
+![add](readme-docs/screenshots/UpdateSampleStatusResult.png)
+
+### Remove Selected Samples
+The only way to deselect samples is to clear the entire "Samples to update" grid of all added samples.
+![add](readme-docs/screenshots/RemoveSampleFromList.png)
+
+This can be easily achieved by clicking the "Clear List" button. 
+![add](readme-docs/screenshots/RemoveSampleFromListResult.png)
```
Comment (jenniferboedker) [inline README.md:101]: I think you can remove the last figure

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**